crates/e2e/tests/e2e/colocation_partial_fill.rs

```diff
@@ -90,7 +90,7 @@ async fn test(web3: Web3) {
         .unwrap();
     assert!(
         // Sell balance is strictly less than 2.0 because of the fee.
-        (1_999_999_000_000_000_000_u128..2_000_000_000_000_000_000_u128)
+        (1_999_000_000_000_000_000_u128..2_000_000_000_000_000_000_u128)
```

Comment:
I imagine this is because gas fees are different on anvil from hardhat (i.e. their `eth_feeHistory` differs slightly).

This is interesting!